component-api/src/main/java/org/talend/sdk/component/api/context/RuntimeContextHolder.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (C) 2006-2022 Talend Inc. - www.talend.com
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.talend.sdk.component.api.context;
+
+import java.util.Map;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+public class RuntimeContextHolder {
```

Comment:
This is used by only Studio, but it's in the common-API. I think that's why the global map was a solution before. to avoid adding a Studio specific code in the common component-runtime API

Reply:
but user need to write code like this in connector, so must in component-api:
```
@RuntimeContext
private RuntimeContextHolder context;
```